nvdaHelper/interfaces/nvdaController/nvdaController.idl

```diff
@@ -62,11 +62,25 @@ typedef [v1_enum] enum {
 } SYMBOL_LEVEL;
 
 /**
- * Allows the controling of NVDA from a remote process
+ * Type signature for a callback that may be implemented by a client.
+ * The callback is called for every mark in the SSML as received by speakSsml when called synchronously.
+ * @param mark The name of the reached mark.

```

Comment:
I find it hard to parse these without a separator value between the parameter name and description. I'm not sure what the standard is, but a colon makes it much more readable in my opinion

```suggestion
 * @param mark: The name of the reached mark.
```

Reply:
Thoughts on this suggestion?

Reply:
My preference would be to keep using the original doc formatting (I.e. with out the colon) so that it remains standard across the codebase. However, I would nto be against changing the syntax  of all c++ code docs to something more readable, as long as tools like doxygen or an equivalent could parse it, and that we do it all in one go.